Google.Api.Gax.Grpc/ClientBuilderBase.cs

```diff
@@ -130,6 +131,18 @@ protected void CopyCommonSettings<TOther>(ClientBuilderBase<TOther> source)
             EmulatorDetection = source.EmulatorDetection;
         }
 
+        /// <summary>
+        /// Copies common settings from the specified builder, expecting that any settings around
+        /// credentials and endpoints will be set by the caller, along with any client-specific settings.
+        /// Emulator detection is not copied.
```

Comment:
nit: just so we remember, maybe add "to avoid infinite recursion on build"

Reply:
Done, thanks.